src/components/calcite-panel/calcite-panel.tsx

```diff
@@ -102,6 +116,19 @@ export class CalcitePanel {
     this.dismissed = true;
   };
 
+  // --------------------------------------------------------------------------
+  //
+  //  Methods
+  //
+  // --------------------------------------------------------------------------
+
+  @Method()
+  async setFocus(focusId?: FocusId) {
+    if (focusId === "dismiss-button") {
```

Comment:
not sure the purpose of this argument. If the only valid value is dismiss-button, why not just have it hardedcoded to target that element and remove the argument altogether?

Reply:
Because we may have the ability to focus a panel in the future.

Reply:
Update: added ability to focus on container.